test/hotspot/jtreg/serviceability/sa/UniqueVtableTest.java

```diff
@@ -31,7 +31,7 @@
  *          jdk.hotspot.agent/sun.jvm.hotspot.types
  *          jdk.hotspot.agent/sun.jvm.hotspot.types.basic
  *
- * @run main/othervm UniqueVtableTest
+ * @run driver UniqueVtableTest
```

Comment:
The other tests you touched in this PR use:
`@run main/othervm ...`
so why did this one have to change to:
`@run driver ...`

Reply:
Due changes in the test it doesn't need to be run in "othervm" mode, "driver" is ok now to make the test a bit faster
I didn't change mode other for other tests